content/docs/getting-started/accounts.md

```diff
@@ -29,9 +29,16 @@ Follow [these instructions to log in on the command line (CLI) and web UI (dashb
 
 <!--<img src="/img/cloud-gov-idp-button-cropped.png" alt="cloud.gov provider button to login page is highlighted in blue" width="373" height="187" />-->
 
-Your cloud.gov account requires multi-factor authentication. To log into the system, you need two "factors" -- something you know (your password) and something you have on your person (your smartphone).
+Your cloud.gov account requires multi-factor authentication: something you know (your password) and something you have on your person (your smartphone). In order to perform multi-factor authentication, you need an authentication application that generates time-based one-time passcodes, such as [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en) or [Authy](https://www.authy.com/app/mobile). When you log into cloud.gov for the first time, follow the instructions to set this up.
```

Comment:
It might make sense to stretch this out to include any Time-based One-time Password Algorithm (e.g.`TOTP`) application.  We had this question come in through support because we recommend `smartphone apps` and they wanted to use `a Google Chrome extension`. They're free to use any TOTP application and there are several Desktop applications as well. [Authy has most platforms covered](https://authy.com/features/multiple-devices/). 

Maybe we can just call it _something you know (your password)_ and _something you have (an MFA/2FA application)_? or do we need to keep the phone part as part of the SSP?